__tests__/integration/multisig.test.js

```diff
@@ -33,29 +32,29 @@ describe('send tx (HTR)', () => {
     minSignatures: 3,
   };
 
-  const addresses = [
-    'wgyUgNjqZ18uYr4YfE2ALW6tP5hd8MumH5',
-    'wbe2eJdyZVimA7nJjmBQnKYJSXmpnpMKgG',
-    'wQQWdSZwp2CEGKsTvvbJ7i8HfHuV2i5QVQ',
-    'wfrtq9cMe1YfixVgSKXQNQ5hjsmR4hpjP6',
-    'wQG7itjdtZBsNTk9TG4f1HrehyQiAEMN18',
-    'wfgSqHUHPtmj2GDy8YfasbPPcFh8L1GPMA',
-    'wgZbCEMHHnhftCAwj7CRBmfi5TgBhfMZbk',
-    'wdz9NeMac7jyVeP2WK4BJWsM1zpd9tgsBb',
-    'wPs7WaRCqwC89uHycLbctDGmWPgH9oZvjp',
-    'wWJJxvr6oSk7WZdE9rpSRMoE6ZqJ3i8VDc',
-    'wbuDJtmM7vg8at2h5o3pTCHE4SASEFYusr',
-    'wPNkywbiw8UHbRQkD3nZ3EHMQsjyTamh9u',
-    'wQBNidXXYpE943BgydUNtarAwNzk612Yip',
-    'wh2eCGzUK9rLThr5D6tyCfckHpBjS97ERA',
-    'wZvajxVp3LabcZiY3XPrivrXiSS6wphRu7',
-    'wgPbL1WzbrEntepHRC92UX6mA2EmaqfDqt',
-    'wbdx4g3rucX3WHmZRXjPEKtRfZ7XSnCGKf',
-    'wiKTnqSN11ukuCWEXRVrRTTPo2mw4fGue3',
-    'wQ4aQP4YqJqfwshLggR2w1Gg3UFhhKhVKs',
-    'wca2xk9S2MVn2UrKh78UScdwXz3xrTp8Ky',
-    'wcUZ6J7t2B1s8bqRYiyuZAftcdCGRSiiau',
-  ];
+  // const addresses = [
```

Comment:
Added an explanation to the comment block on [7ea530c](https://github.com/HathorNetwork/hathor-wallet-headless/pull/174/commits/7ea530c291bb9e49ca7d9b04879ebd0582e448e2).

This variable needed to be removed because of the lint `no-unused-var` rule but, as you said, it's a good reference for developing tests. So instead of removing it, I thought better to keep it as a comment.